### PR TITLE
Support for multiple recipients

### PR DESCRIPTION
**Pull request by scruzin**
This implements Issue #82

### CODE REVIEW

File: notify/notify.go [hunk 1]
```diff
@@ -80,29 +77,33 @@ func (n *Notifier) Init(options ...Option) error {
...
 	if n.publicKey != "" && n.privateKey != "" {
 		clt := mailjet.NewMailjetClient(n.publicKey, n.privateKey)
+		var recipients mailjet.RecipientsV31
+		for _, recipient := range n.recipients {
+			recipients = append(recipients, mailjet.RecipientV31{Email: recipient})
+		}
 		info := []mailjet.InfoMessagesV31{{
 			From:     &mailjet.RecipientV31{Email: n.sender},
-			To:       &mailjet.RecipientsV31{mailjet.RecipientV31{Email: n.recipient}},
+			To:       &recipients,
```
Comment (davidsutts) [inline notify/notify.go:106]: This doesn't look quite right, why don't we still need to use the RecipientsV31 type here?

File: notify/options.go [hunk 2]
```diff
@@ -36,18 +36,31 @@ func WithSender(sender string) Option {
 	}
 }
 
-// WithRecipient sets the recipient email address.
+// WithRecipient sets a single recipient email address.
 func WithRecipient(recipient string) Option {
```
Comment (davidsutts) [inline notify/options.go:40]: Could it make sense to make the WithRecipient option append a recipient rather than set that as the only recipient?

Comment (davidsutts) [reply notify/options.go:40]: or maybe check that it isn't trying to override a previous recipient?

**Review by davidsutts — APPROVED** [review]
I think there might be a few changes to be made, but otherwise LGTM!

File: notify/notify.go (hunk 1 above)
Comment (scruzin) [reply notify/notify.go:106]: RecipientsV31 is nothing more than []RecipientV31, which recipients is.

File: notify/options.go (hunk 2 above)
Comment (scruzin) [reply notify/options.go:40]: Unlike filters, we need a way to override a recipient. The new WithRecipients can be used for multiple recipient.

**Pull request merged**